doc-content/enterprise-only/employee-rostering/optashift-ER-runningwar-proc.adoc

```diff
@@ -25,5 +25,5 @@ EAP_HOME/bin/jboss-cli.sh --connect
 ----
 +
 . Stop the application server.
-. Deploy the `binaries/employee-rostering-webapp-{MAVEN_ARTIFACT_VERSION}.war` directory into the application server and start the application server.
+. Deploy the `binaries/optaweb-employee-rostering-webapp-{MAVEN_ARTIFACT_VERSION}.war` directory into the application server and start the application server.
```

Comment:
I am pretty sure we don't supply a "WAR" file anymore (and the webapp module no longer exists).

Reply:
This file isn't used. It was updated as part of the global update.